tests/test_metrics.py

```diff
@@ -183,7 +181,10 @@ def test(self):
         ret = main(["metrics", "add", "metrics"])
         self.assertEqual(ret, 0)
 
-        ret = main(["metrics", "modify", "-t", "csv", "-x", "0,0", "metrics"])
+        ret = main(["metrics", "modify", "-t", "XSV", "-x", "0,0", "metrics"])
```

Comment:
Let's create a separate test for supported/unsupported types under `tests/unit/repo/metrics`.